lib/esm/ModuleChunkLoadingRuntimeModule.js

```diff
@@ -407,7 +389,7 @@ class ModuleChunkLoadingRuntimeModule extends RuntimeModule {
 									`var loadScript = ${runtimeTemplate.basicFunction(
 										"url, onResolve, onReject",
 										[
-											`return ${importFunctionName}(/* webpackIgnore: true */ url).then(onResolve).catch(onReject)`
```

Comment:
Why remove? If we try to bundle already bundled webpack without it will output an error/warning